play-services-core/src/main/kotlin/org/microg/gms/games/GamesSignInActivity.kt

```diff
@@ -41,19 +48,22 @@ class GamesSignInActivity : AppCompatActivity() {
 
         window.setGravity(popupGravity)
         startActivityForResult(Intent(this, AuthSignInActivity::class.java).apply {
+            Log.d(TAG, "Sign in for $gamePackageName account:${account?.name} scopes:$scopes")
             putExtra("config", SignInConfiguration().apply {
                 packageName = gamePackageName
+                val optionsBuilder =
+                    GoogleSignInOptions.Builder().requestScopes(scopes).requestProfile().requestId().requestEmail()
```

Comment:
Are these really the default scope for Games Sign In? I don't think most Games need or request access to the user's email address or profile.